src/ray/raylet/node_manager.cc

```diff
@@ -636,8 +621,11 @@ void NodeManager::ProcessClientMessage(
     const uint8_t *message_data) {
   auto registered_worker = worker_pool_.GetRegisteredWorker(client);
   auto message_type_value = static_cast<protocol::MessageType>(message_type);
-  RAY_LOG(DEBUG) << "Message of " << protocol::EnumNameMessageType(message_type_value)
-                 << "(" << message_type << ")";
+  RAY_LOG(DEBUG) << "[Worker] Message "
+                 << protocol::EnumNameMessageType(message_type_value) << "("
```

Comment:
Would it be possible to do something similiar at 

https://github.com/ray-project/ray/blob/816406ea3d0eac4c6305aa3a8634bcbc78b06fef/src/ray/common/client_connection.cc#L264-L265

That often prints something like

`ProcessMessage with type 6 took too 1234 ms.` 

which is too opaque.